backend/src/entities/track.entity.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  Entity,
+  PrimaryGeneratedColumn,
+  Column,
+  ManyToOne,
+  OneToOne,
+  JoinColumn,
+} from 'typeorm';
+import { Scene } from './scene.entity';
+import { Song } from './song.entity';
+import { LicenseStatus } from './license-status.entity';
+
+@Entity()
+export class Track {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  startTime: string;
+
+  @Column()
+  endTime: string;
```

Comment:
An improvement could be to add validation for start/end times (e.g., end > start) 👀